src/Plans.jl

```diff
@@ -632,25 +663,82 @@ Display a comparison of two plans, or save it in a text file.
-@inline compare_plans(file::AbstractString, left::Plan, right::Plan; kwargs...) = (
-    open(file, "w") do f
-        compare_plans(f, left, right; kwargs...)
+compare_plans(file::AbstractString, left::Plan, right::Plan; kwargs...) = compare_plans(left, right; outfile=file, kwargs...)
+compare_plans(io::Union{IOBuffer, Base.TTY}, left::Plan, right::Plan; kwargs...) = compare_plans(left, right; io=io, kwargs...)
+
+function compare_plans(left::Plan, right::Plan; outfile = "",
+        io = nothing, 
+        alphabetical=false,  # whether to sort variables
+        pagelines=0,
+        exog_mark="X", endo_mark="~", missing_mark=".", # symbols used for each class
+        delim=" ",  # set to "," to get a CSV file (with 3 skip rows and 1 header row)
+        _name_delim=delim,  # padding after NAME column
+        _range_delim=delim,    # padding between range columns
+        diff = false, # whether to print only variables/shocks which differ
+        summary = false, # whether to print summary information to stdout
+        legend = false # whether to print the legend for the matrix numbers
+    ) #
+    left_vars = keys(left.varshks)
+    right_vars = keys(right.varshks)
+    aligned_right = deepcopy(left)
+    aligned_right.exogenous .= 0
+    copyto!(aligned_right, rangeof(left), right; verbose=false)
+    left_matrix = Int64.(left.exogenous)
+    right_matrix = Int64.(aligned_right.exogenous) .* 2
+    combined_matrix = left_matrix .+ right_matrix
+    combined_mvts = MVTSeries(first(rangeof(left)), collect(keys(left.varshks)), combined_matrix)
+    combined_mvts = combined_mvts[intersect(rangeof(left),rangeof(right)), intersect(left_vars, right_vars)]
+
+    
+
+    if summary || diff
+        differences = combined_mvts .% 3 .== 0
     end
-)
-function compare_plans(io::IO, left::Plan, right::Plan;
-    alphabetical=false,  # whether to sort variables
-    pagelines=0,
-    exog_mark="X", endo_mark="~", missing_mark=".", # symbols used for each class
-    delim=" ",  # set to "," to get a CSV file (with 3 skip rows and 1 header row)
-    _name_delim=delim,  # padding after NAME column
-    _range_delim=delim    # padding between range columns
-)
-    # summary(io, p)
+    if summary
+        if left.range == right.range
+            println("Same range: ", left.range)
+        else
+            println("Range  left: ", left.range)
+            println("Range right: ", right.range)
+        end
+        
+        if Set(left_vars) == Set(right_vars)
+            println("Same variables.")
+        else
+            println("Variables only in left plan: ", setdiff(left_vars, right_vars))
+            println("Variables only in right plan: ", setdiff(right_vars, left_vars))
+            println(length(intersect(left_vars, right_vars)), " common variables.")
+        end
+        for var in keys(combined_mvts)
+            mits = rangeof(combined_mvts[var])[differences[var] .== false]
+            if length(mits) > 0
```

Comment:
`.!differences[var]` may be better than `differences[var] .== false`.  Not sure

Reply:
Done.